doc/CHANGES.md

```diff
@@ -3,28 +3,150 @@
 ## Highlighted Features
 
 * Use consistent colour for admin interface save buttons (Gareth Rees)
+* Remove dependency on Python (Graeme Porteous)
+* Use human-friendly URLs to body listings (Gareth Rees)
+* Add unsubscribe links to survey mailer (Graeme Porteous)
+* Prepare for the Rails 6.1 version (Graeme Porteous)
+* Add script to redact raw emails (Graeme Porteous)
```

Comment:
`s/Add script/Add beta script/` just to signify that its not particularly battle-tested?